src/OrchardCore.Modules/OrchardCore.Sitemaps/Deployment/AllSitemapsDeploymentSource.cs

```diff
@@ -23,7 +29,7 @@ public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlan
 
             var sitemaps = await _sitemapManager.GetSitemapsAsync();
 
-            var jArray = JArray.FromObject(sitemaps);
+            var jArray = JArray.FromObject(sitemaps, _documentJsonSerializerOptions.SerializerOptions);
```

Comment:
There are a lot of `JArray.FromObject()` references in the code. I checked them out for similar omissions but didn't find any.

Reply:
yes I did too. this seems to be the only instance where Polymorphic types are used